### PR TITLE
Update flycap SDK to 2.10.3.169 needed for Ubuntu 16.04

### PR DESCRIPTION
**Pull request by konradb3**
Only tested with Ubuntu 16.04 and ROS Kinetic.

### CODE REVIEW

**Comment by mikepurvis** [comment]
Would you mind updating the Travis job to use trusty/indigo so that it passes for this change?

**Comment by mikepurvis** [comment]
I feel like we're have a lot of breakages and back and forth on which version of flycap to use. I suppose there's not much we can do other than upgrade and wait for screams— I certainly haven't got a full suite of PGR hardware with which to test.

File: .travis.yml [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
   - sudo pip install rosdep
   - sudo rosdep init && rosdep update 
-  - sudo sh -c 'echo "deb http://packages.ros.org/ros/ubuntu precise main" > /etc/apt/sources.list.d/ros-latest.list'
+  - sudo sh -c 'echo "deb http://packages.ros.org/ros/ubuntu trusty main" > /etc/apt/sources.list.d/ros-latest.list'
   - wget http://packages.ros.org/ros.key -O - | sudo apt-key add -
   - sudo apt-get update -qq
 install:
-  - rosdep install --from-paths .. --ignore-src --rosdistro hydro
-  - source /opt/ros/hydro/setup.bash
+  - rosdep install --from-paths .. --ignore-src --rosdistro indigo
```
Comment (mikepurvis) [inline .travis.yml]: You'll need a `-y` here too, to suppress the apt prompt.

**Comment by konradb3** [comment]
The Travis with trusty use custom build version of Python.
It breaks all python dependent deb's including catkin :disappointed:

**Comment by konradb3** [comment]
I can confirm that new flycap don't work on pre16.04 ubuntu.
The new libstdc++ use different ABI, so we need different binaries for new(>=16.04) and old(<16.04) ubuntus. This can be done by branching old version of driver of introducing some magic in cmake to download different flycap based on ubuntu version.

**Comment by mikepurvis** [comment]
Gross.

CMake magic is one way to go; alternatively, we could consider going to split indigo-devel/kinetic-devel branches in the repo, release the Kinetic branch as 0.13.0, and simply manage the change that way.

**Comment by mikepurvis** [comment]
Regarding Python madness, here's a build to check what's going on with Python in the Trusty build environment: https://travis-ci.org/ros-drivers/pointgrey_camera_driver/builds/147197164

Looks like it's not even installed from a debian package at all— maybe just a straight up `make; make install`? Either way, it seems like even under `language: cpp`, Travis still dumps you into a venv: https://docs.travis-ci.com/user/languages/python

**Comment by ironmig** [comment]
@konradb3 Do you have any documentation that says 16.04 is only compatible with flycap SDK 2.10. We're running a submarine with 3 pointgrey cameras using this camera and are having some miscellaneous issues and are wondering if the sdk version might be the problem.

**Comment by konradb3** [comment]
@ironmig On the ptgray (FLIR) website there are two versions of the SDK available for download : 
* FlyCapture 2.9.3.43 SDK - Linux Ubuntu 14.04
* FlyCapture 2.11.3.121 SDK - Linux Ubuntu 16.04

Addition of Ubuntu 16.04 support is probably mentioned in 2.11 release note.

**Comment by tedchina** [comment]
@ironmig @konradb3 I onced tried FlyCapture 2.11.3.121 SDK on Intel Joule with Ubuntu 16.04, the flycap launch was successful and the app window will crash whenever I attempt to turn on the camera. I reported this to FLIR and they confirmed some bug and to release a new version soon.
Before a new release, I was using an older version flycapture2-2.10.3.266-amd64-pkg.tgz, with which this ROS driver can also work successfully.

**Comment by mikepurvis** [comment]
Covered by #127. Thanks all.

**Pull request closed without merging**